.github/workflows/analyze.yml

```diff
@@ -22,7 +22,12 @@ jobs:
     - name: Pack test recipe
       run: |
         pack -r cellpack/tests/recipes/v2/test_spheres.json -c cellpack/tests/packing-configs/test_config.json
-        analyze -a cellpack/tests/analysis-configs/PR_analysis_config.json
+    - name: Modify JSON with PR sub_directory path
+      run: |
+        jq --arg branch_name "$GITHUB_REF_NAME" '.create_report.results_output_path |= "https://cellpack-results.s3.us-west-2.amazonaws.com/\($branch_name)/spheresSST"' cellpack/tests/analysis-configs/PR_analysis_config.json > cellpack/tests/analysis-configs/PR_analysis_config_temp.json
```

Comment:
this is still being interpreted as a string. (check the logs of the action that ran). I would use the GitHub object to get the branch name `https://docs.github.com/en/actions/learn-github-actions/contexts#github-context`. and make sure you're use ${{ }} when needed